src/header/map.rs

```diff
@@ -3218,7 +3220,8 @@ fn usable_capacity(cap: usize) -> usize {
 
 #[inline]
 fn to_raw_capacity(n: usize) -> usize {
-    n + n / 3
+    n.checked_add(n / 3)
+        .expect("overflow while converting to raw capacity")
```

Comment:
similarly, it would be kind of nice (but again, not all that important) if this panic message included the capacity that overflowed:
```suggestion
    match n.checked_add(n / 3) {
        Some(n) => n,
        None => panic!("requested capacity {n} too large: overflow while converting to raw capacity"),
    }
```